### PR TITLE
Add support for for hexadecimal values (closes creditkarma/thrift-parser#2)

### PR DESCRIPTION
**Pull request by phated**

### CODE REVIEW

File: thrift-parser.js [hunk 1]
```diff
@@ -230,6 +265,7 @@ module.exports = (buffer, offset = 0) => {
   };
 
   const readValue = () => readAnyOne(
+    readHexadecimalValue, // This coming before readNumberValue is important, unfortunately
```
Comment (nnance) [inline thrift-parser.js:268]: why is the order important?

**Comment by nnance** [comment]
+1

File: thrift-parser.js (hunk 1 above)
Comment (phated) [reply thrift-parser.js:268]: If `readNumberValue` is attempted before `readHexadecimalValue`, the `0` is parsed as a number and then the parser throws when it reaches `x`.  When `readHexadecimalValue` is attempted first, the `readNumberValue` method never gets attempted if we match `0x`.

**Comment by phated** [comment]
Closed by 8695bfad179883b66a8d29528d15335543cc9c38

**Pull request closed without merging**